sw/device/lib/testing/csrng_testutils.c

```diff
@@ -238,7 +243,114 @@ status_t csrng_testutils_fips_generate_kat(const dif_csrng_t *csrng) {
       .instantiated = true,
       .fips_compliance = false,
   };
-  return csrng_testutils_kat_generate(csrng, 2, kExpectedOutputLen,
+  return csrng_testutils_kat_generate(csrng, 2, kExpectedOutputLen, NULL,
+                                      kExpectedOutput, &kExpectedState);
+}
+
+
+status_t csrng_testutils_fips_instantiate_kat_adata(const dif_csrng_t *csrng,
+                                                    bool fail_expected) {
+  // CTR_DRBG Known-Answer-Tests (KATs).
+  //
+  // Test vector sourced from NIST's CAVP website:
+  // https://csrc.nist.gov/projects/cryptographic-algorithm-validation-program/random-number-generators
```

Comment:
I think it would be valuable to document which file and in particular which test vector (`COUNT`) this is. Maybe you can also add this information for the vector above.